ftw/addressblock/geo/profiles/default_plone5/registry.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<registry>
+
+    <records interface="collective.geo.settings.interfaces.IGeoFeatureStyle">
+        <value key="map_viewlet_position">plone.belowcontentbody</value>
+    </records>
+
+    <records prefix="plone.resources/addressblock-geo-js"
+             interface="Products.CMFPlone.interfaces.IResourceRegistry">
+        <value key="js">++resource++ftw.addressblock.geo/addressblock.js</value>
```

Comment:
I removed the addressblock.js with my PR https://github.com/4teamwork/ftw.addressblock/pull/32/files

Please don't re-add it again ;-)